### PR TITLE
Adds function to create requirement from report

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: toolbox/pkg_check/main.go [hunk 1]
```diff
@@ -202,6 +206,43 @@ func (c *codecovChecker) uploadCoverage() error {
...
+	var sortedPkgs []string
+	for k := range c.codeCoverage {
+		sortedPkgs = append(sortedPkgs, k)
+	}
+
+	sort.Strings(sortedPkgs)
+
+	f, err := os.Create(c.requirement)
+	if err != nil {
+		log.Printf("unable to create fail %s", c.requirement)
```
Comment (yutongz) [inline toolbox/pkg_check/main.go]: typo "fail" -> "file"

File: toolbox/pkg_check/main.go [hunk 2]
```diff
@@ -26,20 +26,24 @@ import (
...
 )
 
 const (
-	jobName      = "JOB_NAME"
-	buildID      = "BUILD_ID"
-	defaultValue = "Default"
+	jobName         = "JOB_NAME"
+	buildID         = "BUILD_ID"
+	defaultValue    = "Default"
+	defaultTreshold = 80.0
```
Comment (yutongz) [inline toolbox/pkg_check/main.go]: Maybe we should get the default from a flag

**Review by yutongz — COMMENTED** [review]
nit

File: toolbox/pkg_check/main.go (hunk 1 above)
Comment (sebastienvas) [reply toolbox/pkg_check/main.go]: done

File: toolbox/pkg_check/main.go (hunk 2 above)
Comment (sebastienvas) [reply toolbox/pkg_check/main.go]: done

**Comment by sebastienvas** [comment]
PTAL

**Review by yutongz — APPROVED** [review]
/lgtm

**Pull request merged**